src/dsp_tools/commands/create/create_on_server/cardinalities.py

```
@@ -87,7 +87,7 @@ def _add_cardinalities_for_one_class(
     problems = []
     for one_card in resource_card.cards:
         if one_card.propname not in successful_props:
-            logger.warning(f"CARDINALITY: Property '{one_card.propname}' not in successes, no cardinality added.")
+            problems.append(UploadProblem(one_card.propname, UploadProblemType.CARDINALITY_PROPERTY_NOT_FOUND))
```

This is the only change in behaviour. I did not want it to fail silently. I was hesitate to spam them but I think it is better to be explicit if it is too much we can always remove it.